src/test/java/com/sap/cloudfoundry/client/facade/ServicesCloudControllerClientIntegrationTest.java

```diff
@@ -457,6 +459,35 @@ void testFetchingOfFailedServiceKey() throws InterruptedException {
         }
     }
 
+    private void deleteServiceKeys(List<CloudServiceKey> serviceKeys) {
+        serviceKeys.parallelStream()
```

Comment:
Execution in parallel is not guarantee

Reply:
Yes, but it is just for the deletion of service keys, so it might delete them a bit faster, and it is for tests only